From 2994386127a32ab0d80874070a0d54abccb454c6 Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Thu, 11 Feb 2016 11:30:36 +0100
Subject: [PATCH 16/17] S2: Change the handling of the handshake control lines
 of UART6

The handshake control lines (RTS/CTS) are handled statically by now.
That means CTS is and input GPIO and RTS is an output gpio which is
driven to low.

Ticket #938
---
 arch/arm/boot/dts/imx6sx-s2.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-s2.dts b/arch/arm/boot/dts/imx6sx-s2.dts
index 778b4bd..72698c2 100644
--- a/arch/arm/boot/dts/imx6sx-s2.dts
+++ b/arch/arm/boot/dts/imx6sx-s2.dts
@@ -240,6 +240,8 @@
 				MX6SX_PAD_GPIO1_IO03__GPIO1_IO_3	0x17059
 				MX6SX_PAD_KEY_COL2__GPIO2_IO_12		0xc9
                 MX6SX_PAD_GPIO1_IO02__GPIO1_IO_2	0xc9
+				MX6SX_PAD_KEY_ROW0__GPIO2_IO_15		0x1b0b1 /* UART_CTS */
+				MX6SX_PAD_KEY_COL0__GPIO2_IO_10		0x1b0b1 /* UART_RTS */
 			>;
         };
 
@@ -323,8 +325,6 @@
 
 		pinctrl_uart6: uart6grp {
 			fsl,pins = <
-				MX6SX_PAD_KEY_ROW0__UART6_CTS_B		0x1b0b1
-				MX6SX_PAD_KEY_COL0__UART6_RTS_B		0x1b0b1
 				MX6SX_PAD_KEY_COL1__UART6_TX		0x1b0b1
 				MX6SX_PAD_KEY_ROW1__UART6_RX		0x1b0b1
 			>;
-- 
1.9.1

